packages/shared/src/extension/popup/components/Settings.tsx

```diff
@@ -95,25 +95,30 @@ export const Settings: React.FC<SettingsProps> = ({
           />
         </ListItem>
       ) : null}
-      {enabled.experimentId ? (
-          <ListItem
-            style={{
-              display: 'flex',
-              flexDirection: 'row',
-              alignItems: 'center',
-            }}
-          >
-            <ListItemIcon>
-              <LabelIcon />
-            </ListItemIcon>
-            <InputLabel>Experiment Id</InputLabel>
-            <Input
-              value={experimentId ?? ""}
-              style={{ marginLeft: 8 }}
-              onChange={doSetExperimentId}
-            />
-          </ListItem>
-        ) : null}
+      {enabled.experimentId &&
+      experimentId &&
+      experimentId !== 'DEFAULT_UNSET' ? (
```

Comment:
We have to get rid off this "DEFAULT_UNSET" value sooner or later and just use `undefined`

Reply:
I will do this today!